src/Microsoft.AspNet.Mvc.ViewFeatures/Rendering/TagBuilder.cs

```diff
@@ -47,8 +55,14 @@ public AttributeDictionary Attributes
             }
         }
 
+        /// <summary>
+        /// Gets or sets the inner HTML value for the element.
```

Comment:
This property is get-only. Also "value" -> "content" since an `IHtmlContentBuilder` can hold lots of values.